### ISSUE TITLE
[Dependencies] - Add a Bot

### ISSUE THREAD

**Issue by deinok** [ckan/ckan#3904]
Take a look at this https://dependabot.com/

It can help upgrading the dependencies.

**Comment by wardi** [comment]
This would be more appropriate after we're off some old dependencies, I think it would be a lot of noise for now

**Comment by deinok** [comment]
The idea is enable this as  before, better, and let the bot upgrade libs that pass the CircleCI.
Most of the problems to be unable to use Python3 its because nobody take this task seriusly

**Comment by wardi** [comment]
If this tool can find all the upgrades that don't break tests then I'm all for it!

As far as Python 3 support you're mistaken. Most of the core development effort is working towards python 3 compatibility right now, including everything tagged https://github.com/ckan/ckan/labels/Pylons%20-%20Flask%20migration

**Comment by deinok** [comment]
@wardi Yeah, the tool only make PR to the repository when they detect unupgraded libs. Somebody must accept that PRs.
Yes, i know that Python3 effort, but even if the source of CKAN is ready for P3 if its libs still P2-only i would be imposible to run it in P3, i think

**Comment by deinok** [comment]
ref: #3309 
As we said, this can be very helpful with the transition.

The bot will only make one PR for dependency.
With a good set of tests, like the CKAN tests, it should be easy to mantain most of the dependencies.
And as I said, I can work on removing and updating libs that are not going to be a simple update of the version number.

**Comment by cclauss** [comment]
> This would be more appropriate after we're off some old dependencies

How do things stand on getting off these old dependencies?    Is there progress on this PR?
* https://requires.io/github/ckan/ckan/requirements/?branch=master

For the bot, https://pyup.io works well for me.

**Comment by frafra** [comment]
The old libraries have been removed. Dependabot can be enabled, at least for security updates and with a low frequency, just to start.

**Comment by amercader** [comment]
@frafra We are almost there! Still a couple of libraries to update https://github.com/ckan/ckan/pull/6709

The repo admins do get alerts about security dependabot updates
